src/main/java/com/checkmarx/intellij/tool/window/results/tree/nodes/ResultNode.java

```diff
@@ -270,79 +335,62 @@ private JPanel getSCAVulnerabilityPathPanel(Result result, JPanel scaBody) {
-        });
-        return aboutVulnerability;
+        scaBody.add(vulnerabilitiesPanel);
     }
 
-    @NotNull
-    private JLabel getSCARemediationLabel(Result result, JPanel scaBody) {
-        JLabel remediationTitle = new JLabel(String.format("<html><b>%s</b></html>", boldLabel(Bundle.message(Resource.REMEDIATION)).getText()));
-        scaBody.add(remediationTitle, "span, growx");
+    /**
+     * Draw SCA references section
+     *
+     * @param scaBody - body panel
+     */
+    private void drawSCAReferences(Result result, JPanel scaBody) {
+        addSectionTitle(scaBody, Resource.REFERENCES);
 
-        int r = JBColor.ORANGE.getRed();
-        int g = JBColor.ORANGE.getGreen();
-        int b = JBColor.ORANGE.getBlue();
-        String hex = String.format("#%02x%02x%02x", r, g, b);
+        JPanel linksPanel = new JPanel();
+        linksPanel.setBorder(JBUI.Borders.empty(-5, 6, 20, 0));
+        linksPanel.setLayout(new MigLayout());
 
-        JLabel remediation = new JLabel();
-        if (result.getData().getRecommendedVersion() != null) {
-            remediation.setIcon(AllIcons.Diff.MagicResolve);
-            remediation.setToolTipText(Bundle.message(Resource.AUTO_REMEDIATION_TOOLTIP));
-            remediation.setText(String.format(Constants.HTML_FONT_YELLOW_FORMAT, hex, "Upgrade to version: " + result.getData().getRecommendedVersion()));
-            remediation.setCursor(new Cursor(Cursor.HAND_CURSOR));
-            if (result.getData().getScaPackageData().isSupportsQuickFix()) {
-                remediation.addMouseListener(new MouseAdapter() {
+        int r = JBColor.BLUE.getRed();
+        int g = JBColor.BLUE.getGreen();
+        int b = JBColor.BLUE.getBlue();
+        String hex = String.format("#%02x%02x%02x", r, g, b);
+        if(result.getData().getScaPackageData() != null && result.getData().getPackageData() != null) {
+            for (int i = 0; i < result.getData().getPackageData().size(); i++) {
+                PackageData packageData = result.getData().getPackageData().get(i);
+                JLabel packageName = new JLabel(String.format(Constants.HTML_FONT_BLUE_FORMAT, hex, result.getData().getPackageData().get(i).getType()));
+                packageName.setCursor(new Cursor(Cursor.HAND_CURSOR));
+                packageName.addMouseListener(new MouseAdapter() {
                     @SneakyThrows
                     @Override
                     public void mouseClicked(MouseEvent e) {
-                        handleGeneralRemediationClick(remediation, result);
+                        Desktop.getDesktop().browse(new URI(packageData.getUrl()));
                     }
                 });
-            } else {
-                remediation.setEnabled(false);
+                linksPanel.add(packageName);
             }
+            scaBody.add(linksPanel);
         } else {
-            remediation.setText("No information");
+            JLabel noInformation = new JLabel("No information");
```

Comment:
This string is repeated, let's add a resource for when we don't have data to display

Reply:
fixed